### PR TITLE
Tripal tools: update to python-tripal 3.1.1

### PR DESCRIPTION
**Pull request by abretaud**
Thanks to @mboudet's work, now supports loading expression data.
These new tools are not tested yet. I have to do it (sigh)

### CODE REVIEW

**Comment by abretaud** [comment]
I've finished testing this, it should be ready for merge now

File: tools/tripal/feature_delete_orphans.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+tripaille feature delete_orphans
+
+&&
+
+echo "Data loaded" > $results
+]]></command>
+    <inputs>
+    </inputs>
+    <outputs>
+        <data format="json" name="results"/>
```
Comment (erasche) [inline tools/tripal/feature_delete_orphans.xml]:
```suggestion
        <data format="txt" name="results"/>
```

File: tools/tripal/expression_sync_biomaterials.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+]]></command>
+    <inputs>
+        <!-- arguments -->
+
+        <!-- options -->
+        <param name="ids" label="Ids" argument="ids" type="text" help="List of ids of biomaterials to be synced (default: all)" />
+
+    </inputs>
+    <outputs>
+        <data format="json" name="results"/>
```
Comment (erasche) [inline tools/tripal/expression_sync_biomaterials.xml]:
```suggestion
        <data format="txt" name="results"/>
```

File: tools/tripal/expression_add_biomaterial.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+echo "Data loaded" > $results
+]]></command>
+    <inputs>
+        <!-- arguments -->
+        <param argument="organism_id" type="select" dynamic_options="list_organisms()" label="Organism" />
+        <param name="file_path" label="Biomaterial description sheet" argument="file_path" type="data" format="xml,tsv,csv" help="A file describing the biomaterial" />
+    </inputs>
+    <outputs>
+        <data format="json" name="results"/>
```
Comment (erasche) [inline tools/tripal/expression_add_biomaterial.xml]:
```suggestion
        <data format="txt" name="results"/>
```

**Pull request merged**